src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -40,6 +40,8 @@ public sealed class VisualStudioInstanceFactory : IDisposable
 
         public VisualStudioInstanceFactory()
         {
+            AppContext.SetSwitch("Switch.System.Diagnostics.IgnorePortablePDBsInStackTraces", false);
```

Comment:
😕 This is not necessary. We target net472 where the default value for this switch is `false`.